developer/community-resources.md

```diff
@@ -0,0 +1,138 @@
+
+[https://github.com/joshuacox/reaction-generic-theme](https://github.com/joshuacox/reaction-generic-theme)
+
+Requirements:
+- Basic LESS/CSS knowledge
+
+Learning content:
+- Customize the visual appearance of Reaction
+- LESS functions helper functions to ease CSS related boilerplate
+
+Kudos for [Joshua Cox!](https://github.com/joshuacox)
+
+
+#### customReactionTheme
+
+Another plugin that is concerned with customizing the Shop's CSS styling, but also shows how to render own markup for different parts of the shop, like the landing page. It does so through usage of Reaction's [component API](http://api.docs.reactioncommerce.com/Components.html)
+
+[https://github.com/curranabell/customReactionTheme](https://github.com/curranabell/customReactionTheme)
+
+
+Prerequisites:
+
+Uses additional meteor package. Open a console in project's root directory and execute this command: `meteor add ultimatejs:tracker-react` to add the required dependency.
+
+
+Requirements:
+- Basic LESS/CSS knowledge
+- Basic React understanding
+
+Learning content:
+- Customize the visual appearance of Reaction
+- Use Reaction's component API to gain control over rendered HTML markup
+
+Mahalo for your kokua, [Curran!](https://github.com/curranabell)
+
+
+#### loanlaux/reaction-hydrotik-plugin
+
+This is an up-to-date fork of hydrotik/reaction-hydrotik-plugin that works with current Reaction version. A visually appealing example on how to customize the landing page of a Reaction shop.
+
+Prerequisites:
+
+- Uses additional npm packages. Open a console in project's root directory and execute these commands: `npm install --save react-slick slick-carousel`
+- Move files in folder <plugin-dir>/public to your root public folder
+
+Requirements:
+- Basic LESS/CSS knowledge
+- Basic React understanding
+
+Learning content:
+- Customize the visual appearance of Reaction
+- Use Reaction's component API to gain control over rendered HTML markup
+- Shows how to integrate third-party React components
+
+Kudos for [Loan!](https://github.com/loanlaux)
+
+#### boomerdigital/reaction-subscription-billing
+
+This plugin is **work-in-progress**, but does address a very common request: A Reaction  plugin that does support the most popular subscription billing features. This is because subscriptions play an important and ever increasing role in the ecommerce landscape.
+
+[https://github.com/boomerdigital/reaction-subscription-billing](https://github.com/boomerdigital/reaction-subscription-billing)
+
+Thank you for helping our ecosystem to grow, [Daniel!](https://github.com/dhonig)
+
+## REST-API for Reaction applications
+DDP is great and good, but ever found yourself in need for a traditional REST API? Here's how to get your API off the ground:
+[https://github.com/kahmali/meteor-restivus](https://github.com/kahmali/meteor-restivus)
+
```

Comment:
also simple:rest https://atmospherejs.com/simple/rest is one we've recommended in the past.

Reply:
Mentioned as alternative.